core/src/main/java/com/sequenceiq/cloudbreak/controller/validation/template/TemplateValidator.java

```diff
@@ -187,16 +197,28 @@ private boolean isComputeInstanceGroup(InstanceGroup instanceGroup) {
         return GROUP_NAME_COMPUTE.equalsIgnoreCase(instanceGroup.getGroupName());
     }
 
+    private boolean isCoordinatorAndExecutorInstanceGroup(Stack stack) {
+        CmTemplateProcessor processor = cmTemplateProcessorFactory.get(stack.getBlueprint().getBlueprintText());
+        Set<ServiceComponent> services = processor.getAllComponents();
+        for (ServiceComponent service : services) {
```

Comment:
maybe a stream + filter would be more elegant here

Reply:
okay will update it.